providers/class.two-factor-dummy.php

```diff
@@ -20,7 +20,7 @@ function authentication_page( $user ) {
 		?>
 		<p><?php esc_html_e( 'Are you really you?', 'two-factor' ); ?></p>
 		<?php
-		submit_button( __( 'Yup.', 'two-factor' ) );
+		submit_button( esc_html__( 'Yup.', 'two-factor' ) );
```

Comment:
As noted below -- https://github.com/georgestephanis/two-factor/pull/4/files#r34511775 -- this shouldn't be escaped here.